packages/slice-machine/pages/onboarding.tsx

```diff
@@ -245,20 +238,19 @@ export default function Onboarding(): JSX.Element {
     return setState({ ...state, videoCompleted: true });
   }
 
-  useEffect(() => {
-    localStorage.setItem(LocalStorageKeys.isOnboarded, "true");
```

Comment:
Could you point me to where you did this again ?
I couldn't find it anymore.
I found the part about the state but couldn't find the part about the local storage.

Reply:
Redux persist handle this right now